### ISSUE TITLE
There is no code files of this repo?

### ISSUE THREAD

**Issue by yangxudong** [xiaoboCASIA/SV-X-Softmax#4]
why this repo is empty.

**Comment by tiandunx** [comment]
I've already implemented SV-X-Softmax and it works fine!

**Comment by LaviLiu** [comment]
@tiandunx Would you like to share your code to github?

**Comment by tiandunx** [comment]
@Laviyy Sorry to reply late. Of course I'm willing to share my implementation.

**Comment by ysc703** [comment]
@tiandunx We are waiting for you! Thanks^_^

**Comment by twmht** [comment]
@tiandunx 

any update?

**Comment by tiandunx** [comment]
Hi all, I'm so sorry for keeping you waiting. I shall release the code after ICCV. Sorry again.  Thanks for keeping an eye upon this paper.

**Comment by luameows** [comment]
@tiandunx Would u mind tell me ur LFW accuracy? I implemented it on mnist, and the classification accuracy was not very good. As for on face recognition, the code is still running. So I cannot get the result now.

**Comment by tiandunx** [comment]
@luameows Sure, on LFW, I achieved 99.866％ using MS only.

**Comment by ysc703** [comment]
> Hi all, I'm so sorry for keeping you waiting. I shall release the code after ICCV. Sorry again. Thanks for keeping an eye upon this paper.

@tiandunx After Nov 3, 2019?  😲😲😲

**Comment by tiandunx** [comment]
At the end of this month.

**Comment by wjgaas** [comment]
@tiandunx  which backbone net did you use to achieve 99.866％ on lfw? res-50? or attention-56?

**Comment by twmht** [comment]
@tiandunx 

What is the value of t you used? And what is the margin loss function you used?

**Comment by tiandunx** [comment]
t = 1.2, margin loss is additive margin softmax@twmht

**Comment by DevilCat** [comment]
> At the end of this month.

Is the code able to be shared? I'll be very appreciated.

**Comment by tiandunx** [comment]
The code is available now.

**Comment by twmht** [comment]
@tiandunx 

thanks for sharing.

So you use all the default values when set `is_am=True` ?

`def __init__(self, feat_dim, num_class, is_am, margin=0.45, mask=1.12, scale=32):`

**Comment by wjgaas** [comment]
@tiandunx Hi, which backbone net did you use to achieve 99.866％ on lfw? res-50? or attention-56?
